### PR TITLE
Added example for Tasmota Two Button Device

### PR DESCRIPTION
**Pull request by cmiguelcabral**
This is a new example for Tasmota, this time using the new integration.
This is an example of how to use two buttons to control one RGB light.

I'm not sure if something is missing, if it is just tell me and I'll update the PR accordingly.

### CODE REVIEW

File: docs/docs/examples/tasmota-double-button.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+In most of my rooms, I have a Sonoff Mini behind the switch in the wall, connected to one or more Zigbee RGB bulbs. Then, I replaced the wall switches with double push buttons of the same brand and model (to keep aesthetics and have my wife's approval). The Sonoff Mini allows connecting a button to the terminal header, plus another button to GPIO16 on the board itself through a pin header used to activate the DIY mode, so I connected those two buttons to the Sonoff so I can control the bulbs without cutting the power when the switch is toggled.
+I use the left button as `Button1`and the right button as `Button2`.
+
+#### Requirements:
+
+Tasmota: v12.2.0 or newer
+
+ControllerX: v4.24.0 or newer
+
+#### This example will:
```
Comment (xaviml) [inline docs/docs/examples/tasmota-double-button.md:24]: No need to add `:` in the title.

File: docs/docs/examples/tasmota-double-button.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+- toggle light on low brightness and warm white on `Button1` single press
+- toggle light on high brightness and warm white on `Button2` single press
+- decrease light brightness when `Button1` is held
+- increase light brightness when `Button2` is held
+- decrease color temperature on `Button1` double pressed
+- increase color temperature on `Button2` double pressed
+- rotate XY color down on `Button1` triple pressed
+- rotate XY color up on `Button2` triple pressed
+
+#### ControllerX apps.yaml example:
```
Comment (xaviml) [inline docs/docs/examples/tasmota-double-button.md:35]: No need to add `:` in the title.

File: docs/docs/examples/tasmota-double-button.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+##### Button1 (decrease):
```
Comment (xaviml) [inline docs/docs/examples/tasmota-double-button.md:37]: No need to add `:` in the title.

File: docs/docs/examples/tasmota-double-button.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      attributes:
+        brightness: 76
+        color_temp: 500
+    DOUBLE: click_colortemp_down
+    TRIPLE: click_xycolor_down
+    HOLD:
+      action: hold_brightness_down
+```
+
+##### Button2 (increase):
```
Comment (xaviml) [inline docs/docs/examples/tasmota-double-button.md:61]: No need to add `:` in the title.

**Comment by cmiguelcabral** [comment]
> Hi @cmiguelcabral ,
> 
> Thanks for the PR, it looks great! Apart from the comment I already left in the existing document, could you also add this page in the navigation: https://github.com/xaviml/controllerx/blob/main/docs/mkdocs.yml#L34
> 
> ```yaml
> - Others:
>           - examples/sonos-display.md
>           - examples/sonos.md
>           - examples/tasmota-switchmode11.md
>           - examples/tasmota-double-button.md
> ```
> 
> In addition, you could also reference this page in the tasmota integration page: https://github.com/xaviml/controllerx/blob/main/docs/docs/start/integrations.md?plain=1#L168
> 
> Regards, Xavi M.

Done!!

**Review by xaviml — APPROVED** [review]

**Comment by xaviml** [comment]
Looks good to me. Thanks @cmiguelcabral :)

**Comment by xaviml** [comment]
Page ready on the netlify page (dev documentation): https://controllerx.netlify.app/controllerx/examples/tasmota-double-button/

**Pull request merged**